test/tests/visibility/visibility_weak.c

```diff
@@ -11,5 +11,5 @@
 
 int main(void)
 {
-	return strcmp(core_version, ROOTSIM_VERSION);
+	return strcmp(core_version, "3.0.0-alpha.3");
```

Comment:
Why don't we use `ROOTSIM_VERSION` here?

Reply:
See previous comment!